src/utils/sources-tree/utils.js

```diff
@@ -34,7 +34,7 @@ export function isInvalidUrl(url: Object, source: Object) {
   return (
     IGNORED_URLS.indexOf(url) != -1 ||
     !source.get("url") ||
-    source.get("loading") ||
+    source.loadedState === "loading" ||
```

Comment:
hmm it looks like the reason that isLoaded isn't working is because it isn't imported